### PR TITLE
Add short id as an internal field

### PR DESCRIPTION
**Pull request by mchv**

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
I thought we decided on not adding this?

**Comment by mchv** [comment]
@LATaylor-guardian Just discovered another tool [that do the extraction](https://github.com/guardian/facia-scala-client/pull/169) so we have now:
- Discussion updater lambda
- Discussion powwow
- Guardian frontend
- Facia client

So it seems to me that providing it as internal may be a good compromise

**Comment by LATaylor-guardian** [comment]
Discussed with @mchv 👍

File: models/src/main/thrift/content/v1.thrift [hunk 1]
```diff
@@ -984,6 +984,11 @@ struct ContentFields {
     40: optional i32 internalContentCode
 
     41: optional bool isLive
+    
+    /*
+     * A short id used as a key for discussion and save for later feature
+     */
+    42: optional string internalShortId
```
Comment (gidsg) [inline models/src/main/thrift/content/v1.thrift]: Does this include the proceeding slash? as Discussion keys are like `/p/12345`

**Comment by johnduffell** [comment]
🎉

File: models/src/main/thrift/content/v1.thrift (hunk 1 above)
Comment (mchv) [inline models/src/main/thrift/content/v1.thrift]: @gidsg yep this is the plan

**Pull request merged**